lib/stock_markets/file_data_processor.rb

```diff
@@ -1,32 +1,61 @@
-require 'json'
+# frozen_string_literal: true
 require 'csv'
 
 module StockMarkets
   class FileDataProcessor
+    LAST_UPDATE_MARKET_MINIMAL_YEARS_COUNT = 2
+    ACTIVE_MARKET_STATUS_NAME = 'ACTIVE'
+    MARKET_NAME_CSV_COLUMN = :nameinstitution_description
 
-    attr_accessor :format, :data
+    attr_accessor :data
 
     def initialize
-      @format = 'json'
       @data = []
+      @parsed_csv = nil
     end
 
-    def load_from_disk
-      self.data = CSV.table(File.expand_path("../../data/stock_markets.csv", __FILE__))
+    def load_from_disk!
+      self.parsed_csv = CSV.table(StockMarkets.configuration.data_file_path)
```

Comment:
what happens if `self.parsed_csv` will not be assigned to data for any reason. looks like the var will be `nil` and we get an exception into `transform_to_hash` method.

Reply:
Great catch. I will move it to initializer